docs/integrations/spark.md

```diff
@@ -389,7 +397,32 @@ On Hadoop 2, configure:
 </configuration>
 ```
 
-(You can also set a default output committer using mapred.output.committer.class and possibly setting mapred.mapper.new-api=false.)
+The lakeFS output committer holds Hadoop working objects on temporary
+branches and merges them back to the output branch when done.  Objects
```

Comment:
Consider:
```suggestion
The lakeFS output committer holds Hadoop working objects on temporary lakeFS 
branches and merges them back to the destination location when done.
```

Reply:
Thanks!  Used your change and changed it further to focus even more on branches.